docs/body.md

```diff
@@ -148,13 +148,16 @@ Used when openstack processes are running inside of docker containers. Provide t
 ### Datasources 
 - CinderServiceStatusDataSource: Checks the status of Cinder services via the Cinder API. 
 - EventsAMQPDataSource: Stores events received from OpenStack Ceilometer via AMQP.
-- HeartbeatsAMQPDataSource: Checks that heartbeats are received from OpenStack Ceilometer via AMQP. 
 - NeutronAgentStatusDataSource: Checks the status of Neutron agents via the Neutron API. 
 - NovaServiceStatusDataSource: Checks the status of Nova services via the Nova API. 
 - PerfAMQPDataSource: Stores performance data received from OpenStack Ceilometer via AMQP. 
 - PerfCeilometerAPIDataSource: Used to capture datapoints from OpenStack Ceilometer. 
 - QueueSizeDataSource: Checks the number of unprocessed messages in Ceilometer AMQP queues.
 
+NOTE: As of Pike, HeartBeats are no longer supported by OpenStack, and so the
+      Heartbeat datasource was removed. Users will be able to verify that
+      OpenStack is active by monitoring data collection directly.
```

Comment:
Just a few thoughts on the phrasing..  

As of the Pike (Late 2017) OpenStack release, the dispatcher mechanism previously used by this zenpack is no longer supported by ceilometer.  Because of this, the "heartbeat" mechanism that was previously used to verify connectivity between ceilometer and Zenoss is no longer possible, and /Status/Heartbeat events will not be created if Zenoss stops receiving data from ceilometer.

Connectivity problems between ceilometer and Zenoss will still be reported in the ceilometer agent-notification.log file on the OpenStack hosts.